pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -985,3 +985,49 @@ func reformatSecretConfigUnderscoreWithDash(secretConfig types.ServiceSecretConf
 
 	return newSecretConfig
 }
+
+//	looks for an initContainer resources and its passed as labels
+//
+// if there is no image, it does not fill the initContainer
+// https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
```

Comment:
Let's conform with the Golang comments style here. I suggest:
```suggestion
// fillInitContainers looks for an initContainer resources and its passed as labels
// if there is no image, it does not fill the initContainer
// https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
```